packages/@lwc/engine-server/src/__tests__/fixtures/render-dynamic-value/modules/x/component/component.js

```diff
@@ -0,0 +1,17 @@
+import { LightningElement } from 'lwc';
+
+export default class extends LightningElement {
+  isUndefined = undefined;
+  isNull = null;
+  isFalse = false;
+  isTrue = true;
+  isZero = 0;
+  isNegZero = -0;
+  isNaN = NaN;
+  isInfinity = Infinity;
+  isNegInfinity = -Infinity;
+  isEmptyString = '';
+  isSymbol = Symbol('yolo')
+  isArray = ['foo', 'bar']
+  isObject = {foo: 'bar', baz: 'quux'}
+}
```

Comment:
```suggestion
  isNullObject = Object.create(null)
  isTrickyArray = [Object.create(null)]
  isFunction = function whoWouldDoThis() { return 'somebody, probably!' }
  isString = 'ooo eee, ooo ah ah, ting tang, walla walla, bing bang'
  isNumber = 6.02214076e23
}
```

Reply:
What if we get real exotic
```kotlin
isStringObject = new String('but why tho')
isBooleanObject = new Boolean(false)
```

Reply:
```scala
isRecursiveObject = (x => { x.x = x; return x })({})
```

Reply:
Let's not go nuts 😛

Reply:
![booo](https://media1.giphy.com/media/v1.Y2lkPTc5MGI3NjExb2pkZzQzbXM3MGdzejcxbjFzYW96d2hrbDhyOGZlM2tzcGF4eGl6eCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/6IGNW4wiyU8Mw/giphy.webp)

Reply:
I don't get zoomer memes, sorry

Reply:
![funfetti](https://media1.giphy.com/media/v1.Y2lkPTc5MGI3NjExejgwN2Y4aWU4MDJnbXk4aWV3Y3NtaW11cWt2Y3p0OWRkYXBkYmx6OCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/EhwFIP584mIBa/giphy.webp)